src/main/io/ledstrip.h

```diff
@@ -73,4 +73,5 @@ bool parseColor(uint8_t index, const char *colorConfig);
 void applyDefaultColors(hsvColor_t *colors, uint8_t colorCount);
 
 void ledStripEnable(void);
+void ledStripDisable(void);
```

Comment:
since `ledStripDisable` is only called in `ledstrip.c` it can be declared `static` and you can remove the declation from `ledstrip.h`